test-projects/demo/dashboards/welcome.visivo.yml

```diff
@@ -71,9 +71,14 @@ dashboards:
           - width: 2
             table:
               name: Fibonacci Waterfall Table
-              trace: ref(Fibonacci Waterfall Trace)
+              traces:
+                - ref(Fibonacci Waterfall Trace)
               columns:
-                - header: Widget Type
-                  column: x_data
-                - header: Widget Sales
-                  column: y_data
+                - trace_name: Fibonacci Waterfall Trace
+                  column_defs:
+                    - header: Widget Type
```

Comment:
We should also make the header optional and just parse the key name pulling out the last value splitting by the dot, removing underscores and capitalizing ie `key.split('.')[-1].replace('_', '').capitalize`

Reply:
So in this example if you didn't specify a header you would get `X Data`